webpack.config.js

```diff
@@ -1,6 +1,6 @@
-import path from 'path';
-import HtmlWebpackPlugin from 'html-webpack-plugin';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+const path = require ('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
This looks fine to me as we saw the linter was working.
But it's much better if we use es6 syntax for import but this is also fine with me.

Reply:
the ES6 import was causing errors in Heroku deployment, saying can use import outside a module